Templates/SwiftUI App.xctemplate/ExampleFlowCoordinator.swift

```diff
@@ -40,11 +38,8 @@ final class ExampleFlowCoordinator: NavigationStackCoordinator {
 }
 
 extension ExampleFlowCoordinator {
+    @EnumIdentable
     enum Destination: String, Hashable, Identifiable {
```

Comment:
```suggestion
    enum Destination: Hashable, Identifiable {
```

We also might make this macro add the conformances, as previously discussed. 

https://github.com/futuredapp/futured-macros/pull/1